native/jni/magiskhide/hide_policy.cpp

```diff
@@ -35,6 +35,10 @@ void hide_unmount(int pid) {
 
     // Unmount dummy skeletons and MAGISKTMP
     targets.push_back(MAGISKTMP);
+    auto magiskpts = MAGISKTMP + "/" SHELLPTS;
```

Comment:
This should already be handled by unmounting MAGISKTMP, shouldn't it?

Reply:
No. If `magiskpts` is not unmounted, `MAGISKTMP` cannot be mounted with `target is busy.`